src/App.tsx

```diff
@@ -10,22 +10,34 @@ import { MyTimer } from './components/Timer';
 import { usePrevious } from './hooks/usePrevious';
 import MyButton from './components/MyButton';
 import { useMyCallback } from './hooks/useMyCallback';
+import UserData from './mock/userdata';
+import { User } from './models/user';
+import MyList from './components/MyList/MyList';
 
 function App() {
   const [showTime, setShowTime] = useState(true);
   const previousShowTime = usePrevious(showTime);
   const btnContent = useMemo(() => <>show time</>, []);
+  const [userData, setUserData] = useState<User[]>([]);
 
   const changeShowTime = useMyCallback(() => {
     setShowTime((prevValue) => !prevValue);
   }, []);
 
+  useEffect(() => {
+    // Fake fetching
+    setTimeout(() => {
+      setUserData(UserData);
+    }, 800);
+  });
+
   return (
     <div>
       {String(showTime)} / {String(previousShowTime)}
       <br />
```

Comment:
nit: using `<br />` for margin spacing will probably make you look bad, we have CSS for that, `<br />` is to break text

Reply:
Removed, I used this since at the time I was just fiddling aroung with react and didn't want to touch css